Makefile

```diff
@@ -1,15 +1,24 @@
 GO := GOFLAGS="-mod=vendor" go
-CMDS := $(addprefix bin/, $(shell ls ./cmd))
+CMDS := $(addprefix bin/, $(shell ls ./cmd | grep -v opm))
+OPM := $(addprefix bin/, opm)
 SPECIFIC_UNIT_TEST := $(if $(TEST),-run $(TEST),)
+PKG := github.com/operator-framework/operator-registry
+GIT_COMMIT := $(shell git rev-parse HEAD)
```

Comment:
Can we use `git rev-parse --short HEAD` here? The quay builder we use for our automation has a nice feature that can automatically tag container images with the short commit hash.

Reply:
Sounds good. Fixed.